pkgs/applications/version-management/gogs/default.nix.patch

```diff
@@ -0,0 +1,22 @@
+2,3c2
```

Comment:
Thanks a lot @cstrahan, I think it's possible to override the go2nix output to add the `postInstall` hook, and tweak the `buildFlags`, it's indeed a lot cleaner than patching `default.nix`.

However the conditional `sqliteSupport` is a bit trickier, since it needs to remove an entry from `extraSrcs` if `sqliteSupport` is `false`, and `extraSrcs` is an [argument to `buildGoPackage`](https://github.com/NixOS/nixpkgs/blob/37d1dcd12d1367a9b0be23947ffe8a0539c2f2b7/pkgs/development/go-modules/generic/default.nix#L18), but not an attribute of the actual derivation, so I think that `lib.overrideDerivation` will not help. I can just ignore this part, it will result in downloading an unnecessary Go package if `sqliteSupport` is `false` which is not that bad.

But if anyone have an idea how to gracefully handle this with overriding, I would happily take it. :smile: